database/migrations/20191220155237_withdrawal_fiat/postgres.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+CREATE TABLE IF NOT EXISTS withdrawal_fiat
+(
+    id bigserial PRIMARY KEY NOT NULL,
+    withdrawal_fiat_id        uuid REFERENCES withdrawal_history(id) ON DELETE CASCADE,
```

Comment:
What are your thoughts on `ON DELETE RESTRICT` to prevent the deletion of records?
I also feel that cascading deletion should go down rather than up (or is this how it works with postgres?)

Reply:
yeah i was unsure if we should ever remove data when i was first creating the schema for this

if you delete the withdrawal_history record it will cascade and delete any withdrawal_fiat or withdrawal_crypto record that match since you shouldn't have either of these without a withdrawal_history record but yu can the other way around

Guess we will need to discuss what is best base line to have and users can customise as required

@thrasher-  cascade or no cascade?

Reply:
Am a fan of no deletions considering the nature of this info. For tax reporting and accountability of trades/portfolio we'll need the entire history without any records missing/deleted